README.md

```diff
@@ -0,0 +1,282 @@
+[](https://learn-2.galvanize.com/ "Galvanize Logo navigates to curriculum overview")
+
+Cognizant - Java CND S2 - 2021-02-22 - Remote
+
+-   [Cognizant - Java CND S2 - 2021-02-22 -
+    Remote](https://learn-2.galvanize.com/cohorts/2560 "Cognizant - Java CND S2 - 2021-02-22 - Remote")
+
+2
+
+CD
+
+Christopher Diehl
+
+-   [My Account](https://auth.galvanize.com/account)
+    christopher.diehl@cognizant.com
+-   [Sign Out](https://learn-2.galvanize.com/sign_out)
```

Comment:
Was it intentional to include the header from the Galvanize Learn page?

Also, nice copy/paste into README from the HTML, I'm impressed with the parsing.